nih-sparc/sparc-design-system-components
# feat(Typography): Documentation cu-guunp5

# Description

The purpose of this PR is to add documentation for the typography

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/1493195/101688670-56943600-3a3a-11eb-9d10-86d62feae619.png">

## Type of change


# How Has This Been Tested?

- Start the storybook
- Go to components/typography
- All docs should match [the designs](https://app.abstract.com/share/e17695cc-fc3c-4007-8814-4012547a2efe?collectionId=ed37ad59-75c5-496f-8bc4-2b7066d0bbd9&collectionLayerId=9b64e444-0bb1-4b7c-8f01-a75b314ad345&present=true&preview=false&sha=521373b6eedaaefbb99f1c39c5e56cb3bdd74c3e)
- The docs section should have the additional copy

# Checklist:
